README.md

```diff
@@ -2,3 +2,12 @@
 
 `<paper-tooltip>` is a Material Design label that appears when the user hovers
 over an element with the cursor, or focuses the element with the keyboard.
+
+If the target is focusable, the focus and blur event are preferred over the mouse-events.
+So if the cursor is still in an input field, the tooltip stays althought the mouse was moved elsewhere.
+
+## Attributes
```

Comment:
Ok, but where? There are neither focus nor blur properties in that section 'cause we don't need these as props. I reckon it's a plain helpful description of the general functionality...